Dockerfile

```diff
@@ -0,0 +1,33 @@
+# ==========================================
+FROM helsinkitest/node:14-slim as deployable
+# ==========================================
+# Hardcode $HOME to /app, as yarn will need to put some stray files inside $HOME
+# In Openshift $HOME would be / by default
+ENV HOME /app
+# Most files from source tree are needed at runtime
+COPY  . .
```

Comment:
Double space between `COPY` and `.`. Not the most serious of offenses, but happened to catch my eyes 🙂

Reply:
One of the issues Sonarcloud is complaining about, fixed on next commit which is focused on fixing Sonarcloud issues.